brianlong/www.validators.app
# 177602946 scores with null ip address and network

#### What's this PR do?
Validators that are delinquent for too long or have 0 stake should not be counted in stats

#### How should this be manually tested?
 - run daemons gather_rpc_mainnet_daemon.rb and validator_score_mainnet_v1_daemon.rb,
 - run sidekiq worker workers/validator_check_active_worker.rb
 - check if is_active and is_rpc flags are updated as expected
 - check if the scores are right

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/177602946

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
